### ISSUE TITLE
When will we have answers?

### ISSUE THREAD

**Issue by kitgary** [MindorksOpenSource/android-interview-questions#4]
This is really awesome! It really helps to prepare for the Android developer interview. When will we have answers? Some questions are hard or too general to me, it's difficult to find the answer.

Thanks

**Comment by dharan1011** [comment]
Listed topics are well described enough to get through interview. But i don't think we Will be having answers for those topics rather you as a Developer need to go through them, understand and solve the problem. 
Just go through below links for better understanding of concepts
https://blog.mindorks.com/tagged/android
https://guides.codepath.com/android
http://www.vogella.com/tutorials/android.html

**Comment by amitshekhariitbhu** [comment]
@kitgary We will try to add all the answers very soon.

**Comment by ashtrisk** [comment]
Hi @amitshekhariitbhu, as @dharan1011 said, by showing curiosity and may be searching for answers on our own will help us in gaining more understanding of the concepts. May be.

**Comment by dharan1011** [comment]
@ashtrisk :+1: But still it will be better contributing useful resources link (answers in this context) to repo will help for better understanding.

**Comment by AdityaAnand1** [comment]
Surely there's a more efficient way than making every android developer google the answer to all these questions. Definitely, we should individually learn on our own and discover these things in the process of making apps but I feel like this repo would offer far more value if it had answers to these questions right here. Even as an experienced developer, I'd still love to have a resource where I can find definitive, well structured and easily conveyed answers to these questions.

Links become dead. The things in those links becomes outdated and those things were in the first place, probably written by a single person. Having these questions answered in a github repo would offer us a chance to collaborate and curate the answers to actually help the reader not only for a technical interview but to become a better developer.

Just my two cents :)

**Comment by amitshekhariitbhu** [comment]
@AdityaAnand1 Yes, sure we will add all the answers at a place where everyone can edit and suggest better answers.

**Comment by ashtrisk** [comment]
@AdityaAnand1 I do agree with you :). And if the answers are with code examples they will be better.

**Comment by Ahmed9914** [comment]
> @AdityaAnand1 Yes, sure we will add all the answers at a place where everyone can edit and suggest better answers.

So, does this mean that contribution here with answers for unanswered question not available?

**Comment by AdityaAnand1** [comment]
Any updates?

**Comment by adarshgumashta** [comment]
Cam we have any updates on this ?

**Comment by aershov24** [comment]
Answered and categorised by difficulty on [www.fullstack.cafe/Android](https://www.fullstack.cafe/Android)

**Comment by amitshekhariitbhu** [comment]
We have added many answers. We will keep adding more.
